Finmer.Core/Serialization/FurballFileDeviceBinary.cs

```diff
@@ -36,22 +37,19 @@ public override Furball ReadModule(FileInfo file)
                         Metadata = ReadHeaderFromStream(instream)
```

Comment:
Reading only the magic and format version was what I originally had in mind back when I originally thought of all this, but since that requires splitting up the furball metadata reading I decided to (three months ago) use a simpler approach. I'll look into changing this.